docs/pages/api-reference/operators.md

```diff
@@ -130,9 +130,24 @@ Fennel does not allow you to drop keys or timestamp columns from a dataset.
 
 <pre snippet="api-reference/operators_ref#drop"></pre>
 
+
+### Dropnull
+
+Fennel allows you to drop null values from columns of a dataset using the `dropnull` operator.
+
+The `dropnull` operator has the following parameters:
+
+1. `columns: List[str]` - positional argument, that specifies the list of columns to apply dropnull on from the dataset.
```

Comment:
Can user leave arguments to `dropnull` empty? If so, worth documenting that here?

Reply:
We should also mention that this operator could take an actual list of names or simply names without list literal (not exactly like this, hopefully in better language). Also, please do the same for select and drop if we haven't already.